### DISCUSSION TITLE
How to customize the image displayer similar to instagram?

### DISCUSSION THREAD

**Discussion by chuvanhoang888** [fluttercandies/flutter_wechat_assets_picker#356] [Ideas]
Hello Sir, can you help me, How to customize the image displayer similar to Instagram Photo Picker?
![z3697947694619_3e6115bef6e3a0a7cccb366c1772d2d8](https://user-images.githubusercontent.com/104492264/188429017-717403b2-de82-47fc-8ab1-967a3d2c78e8.jpg)
![z3697947696266_ecc327c99dd8fc115c80f5bdc61ce8a5](https://user-images.githubusercontent.com/104492264/188429025-11ab8ba1-95b4-442b-83a6-3fb10c2ee168.jpg)

**Comment by AlexV525** [comment]
Use `insta_assets_picker`.
